release-notes/swan-lake-release-notes/swan-lake-2201.2.0-release-note.md

```diff
@@ -196,13 +196,24 @@ digraph "org/package:0.1.0" {
 
 #### New features
 
+##### Language Server
+
+- Added API docs reference support on hover
+- Added new code actions to extract anonymous records into records and to generate undefined record types
+- Introduced new code actions to generate getters and setters for class-level variables
+- Added a new code action to make annotation declarations with the 'source' attach point(s) constant
+
 ##### OpenAPI Tool
 Added support for generating client resource methods in the client generation command. The preferred client method type can be chosen using the `--client-methods=<remote(default)|resource>` option.
   - `$ bal openapi -i <OpenAPI contract> --client-methods=resource`
   - `$ bal openapi -i <OpenAPI contract> --mode client --client-methods=resource`
 
 #### Improvements
 
+##### Language Server
+- Improved the `Create variable` code action in async send action
+- Added completion support for resource access action context
```

Comment:
```suggestion
- Added completion support for the `resource access` action context
```

Reply:
@mindula, please refer to this.
https://github.com/ballerina-platform/ballerina-release/pull/2094#discussion_r946328832

Reply:
Fixed in #2095